MovieQuiz/Presentation/MovieQuizViewController.swift

```diff
@@ -1,72 +1,101 @@
 import UIKit
 
 final class MovieQuizViewController: UIViewController {
-    // MARK: - Lifecycle
+    
+    //MARK: - IBOutlet
+
+    @IBOutlet weak private var counterLabel: UILabel!
+    @IBOutlet weak private var textLabel: UILabel!
+    @IBOutlet weak private var imageView: UIImageView!
+    
+    // MARK: - Private Properties
+    
+    private let questions = QuizQuestion.questions
+    private var currentQuestionsIndex = 0
+    private var score = 0
+    private  var convertStepViewModel: QuizStepViewModel {
+            convertToQuizStepViewModel(model: questions[currentQuestionsIndex])
+    }
```

Comment:
Соблюдайте принятые правила форматирования, можно также выделять код и зажимать control + i
```suggestion
    private var convertStepViewModel: QuizStepViewModel {
        convertToQuizStepViewModel(model: questions[currentQuestionsIndex])
    }
```